src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,48 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.DriverServiceImpl;
+import mate.jdbc.service.ManufacturerService;
+import mate.jdbc.service.ManufacturerServiceImpl;
+
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        System.out.println("Magic STARTS...");
+
+        ManufacturerService manufacturerService = new ManufacturerServiceImpl();
+        DriverService driverService = new DriverServiceImpl();
```

Comment:
You should get service implementation via Injector class. Fix it in main method.